### PR TITLE
Fix/duplicate erv

### PR DESCRIPTION
**Pull request by mdahlhausen**

### CODE REVIEW

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.prototype_inputs.json [hunk 1]
```diff
@@ -1050,7 +1050,7 @@
       "exterior_fuel_equipment1_power": 244443.956043956,
       "exterior_fuel_equipment1_schedule": "Large Office BLDG ELEVATORS",
       "exterior_fuel_equipment2_power": 1270.8,
-      "exterior_fuel_equipment2_schedule": "OfficeLarge ELEV_LIGHT_FAN_SCH_ADD_DF",
+      "exterior_fuel_equipment2_schedule": null,
```
Comment (lymereJ) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.prototype_inputs.json:1053]: @leijerry888 - Is that in line with the elevator changes you've made?

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.prototype_inputs.json [hunk 2]
```diff
@@ -2462,7 +2462,7 @@
       "chiller_capacity_guess": null,
       "pszac_cooling_type": "Single Speed Heat Pump",
       "pszac_heating_type": "Single Speed Heat Pump",
-      "pszac_fan_type": "Cycling",
+      "pszac_fan_type": "ConstantVolume",
```
Comment (lymereJ) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.prototype_inputs.json:2465]: @dmaddoxwhite - Is that in line with the unitary system changes you made?

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.prototype_inputs.json (hunk 1 above)
Comment (leijerry888) [reply lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.prototype_inputs.json:1053]: @lymereJ 
No. For 2004 large office, I changed from 

      "exterior_fuel_equipment2_power": 1270.8,
      "exterior_fuel_equipment2_schedule": "OfficeLarge ELEV_LIGHT_FAN_SCH_ADD_DF",

to

      "exterior_fuel_equipment2_power": 1942.8,
      "exterior_fuel_equipment2_schedule": "OfficeLarge ELEV_LIGHT_FAN_SCH_24_7",

see #844

Comment (lymereJ) [reply lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.prototype_inputs.json:1053]: This is for the detailed large office building, so we're good.

**Comment by mdahlhausen** [comment]
@lymereJ @dmaddoxwhite @leijerry888 I reverted the changes to the prototype input files.  That should be a separate branch; I thought it would be easy to include here but apparently not.

Once this branch is merged, can you rebuild the .json from the spreadsheets on master and resolve any conflicts/changes?

**Comment by lymereJ** [comment]
Sounds good!

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.prototype_inputs.json (hunk 2 above)
Comment (dmaddoxwhite) [reply lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.prototype_inputs.json]: I think this should work, and it is more consistent with the DOE small office prototype, which does not have cycling fans.

**Pull request merged**